dev-packages/size-limit-gh-action/index.mjs

```diff
@@ -271,7 +262,22 @@ async function run() {
       isNaN(thresholdNumber) || limit.hasSizeChanges(base, current, thresholdNumber) || sizeLimitComment;
 
     if (shouldComment) {
-      const body = [SIZE_LIMIT_HEADING, markdownTable(limit.formatResults(base, current))].join('\r\n');
+      const bodyParts = [SIZE_LIMIT_HEADING];
+
+      if (baseIsNotLatest) {
+        bodyParts.push(
+          '⚠️ **Warning:** Base artifact is not the latest one, because the latest workflow run is not done yet. This may lead to incorrect results.',
```

Comment:
Same as above: How about adding a line of what could be done to get correct results?